.github/workflows/release.yml

```diff
@@ -30,21 +27,22 @@ jobs:
       - name: Checkout
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
         with:
+          token: ${{ secrets.TF_DEVEX_COMMIT_GITHUB_TOKEN }}
```

Comment:
I think it'd be really good to avoid using this secret except the last possible moment for pushing to Git. Most things should be able to use the automatic token. This would prevent possible exfiltration should one of the called actions become compromised.